csharp/src/Ice/IObject.cs

```diff
@@ -163,10 +163,10 @@ private static Ice.OutputStream WriteParamEncaps(byte[]? v, bool ok, Current cur
                 return Protocol.CreateEmptyResponseFrame(current);
             }
 
-            if (!ok)
-            {
-                current.DispatchObserver?.UserException();
-            }
+            // if (!ok)
```

Comment:
why is this commented out?

Reply:
Current no longer carries the dispatch observer and the BlobjectAsync will be removed entirely very soon.